cts-java/src/test/java/org/ga4gh/cts/api/g2p/GenotypePhenotypeSearchIT.java

```diff
@@ -0,0 +1,413 @@
+import org.ga4gh.cts.api.TestData;
+import org.ga4gh.cts.api.Utils;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.net.HttpURLConnection;
+
+import ga4gh.GenotypePhenotype;
+import ga4gh.GenotypePhenotypeServiceOuterClass;
+import ga4gh.GenotypePhenotypeServiceOuterClass.EvidenceQuery;
+import ga4gh.GenotypePhenotypeServiceOuterClass.SearchGenotypePhenotypeRequest;
+import ga4gh.GenotypePhenotypeServiceOuterClass.SearchGenotypePhenotypeResponse;
+import ga4gh.GenotypePhenotypeServiceOuterClass.SearchPhenotypesRequest;
+import ga4gh.GenotypePhenotypeServiceOuterClass.SearchPhenotypesResponse;
+import ga4gh.Metadata.OntologyTerm;
+import ga4gh.SequenceAnnotationServiceOuterClass;
+import ga4gh.SequenceAnnotationServiceOuterClass.SearchFeaturesRequest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.ga4gh.cts.api.Utils.catchGAWrapperException;
+
+/**
+ * Tests dealing with searching for GenotypePhenotype associations.
+ *
+ * @author Brian Walsh
+ */
+@Category(GenotypePhenotypeTests.class)
+public class GenotypePhenotypeSearchIT implements CtkLogs {
+
+    private static Client client = new Client(URLMAPPING.getInstance());
+
+    /**
+     * Ensure an error is thrown if no parameters are supplied.
+     *
+     * @throws GAWrapperException if the server finds the request invalid in some way
+     * @throws UnirestException if there's a problem speaking HTTP to the server
+     * @throws InvalidProtocolBufferException if there's a problem processing the JSON response from the server
+     */
+    @Test
+    public void ensureCatchNoParametersSearch() throws InvalidProtocolBufferException, UnirestException, GAWrapperException {
+        SearchGenotypePhenotypeRequest request = SearchGenotypePhenotypeRequest
+                .newBuilder()
+                .build();
+        final GAWrapperException didThrow =
+                catchGAWrapperException(() -> {
+                    client.genotypePhenotype.searchGenotypePhenotypes(request);
+                });
+        assertThat(didThrow.getHttpStatusCode()).isEqualTo(HttpURLConnection.HTTP_NOT_FOUND);
+    }
+
+     /**
+      * Simple search for feature, searches by Genomic feature name.
+      * @throws GAWrapperException if the server finds the request invalid in some way
+      * @throws UnirestException if there's a problem speaking HTTP to the server
+      * @throws InvalidProtocolBufferException if there's a problem processing the JSON response from the server
+      */
+     @Test
+     public void testGenotypesSearchByNameKIT() throws   InvalidProtocolBufferException, UnirestException, GAWrapperException  {
+         final String phenotypeAssociationSetId = Utils.getPhenotypeAssociationSetId(client);
+         SearchFeaturesRequest request = SearchFeaturesRequest
+                 .newBuilder()
+                 .setFeatureSetId(Utils.getFeatureG2PSetId(client))
+                 .setName(TestData.FEATURE_NAME)
```

Comment:
@david4096 sure.  should I make the change to the comments at:
https://github.com/ga4gh/schemas/blob/master/src/main/proto/ga4gh/sequence_annotation_service.proto#L91 .
https://github.com/ga4gh/schemas/blob/master/src/main/proto/ga4gh/sequence_annotation_service.proto#L98